src/pages/index.tsx

```diff
@@ -103,6 +103,15 @@ interface Storefront {
   ownerAddress: string
 }
 
+export interface Marketplace {
+  subdomain: string,
+  name: string,
+  description: string,
+  logoUrl: string,
+  bannerUrl: string,
+  auctionHouseAddress: string
```

Comment:
```suggestion
  auctionHouse: AuctionHouse
```

Need to add interface definition for AuctionHouse.